model/usersModel.ts

```diff
@@ -0,0 +1,22 @@
+export const joinQuery = ({
+  uid,
+  nickname,
+  myTeam,
+}: {
+  uid: string;
+  nickname: string;
+  myTeam: string;
+}): [string, any[]] => {
+  const sql = `
+        INSERT INTO "BaseballArchive".users (uid, nickname, my_team) 
+        VALUES ($1, $2, $3);
+      `;
+  const values = [uid, nickname, myTeam];
+  return [sql, values];
+};
+
+export const getQuery = ({ uid }: { uid: string }): [string, any[]] => {
```

Comment:
이런 네이밍이면 더 좋을 것 같네요!
```suggestion
export const getUserQuery = ({ uid }: { uid: string }): [string, any[]] => {
```